### PR TITLE
Announcement for Bazel 0.21

### PR DESCRIPTION
**Pull request by laurentlb**

### CODE REVIEW

File: _posts/2018-12-19-bazel-0.21.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+layout: posts
+title: Bazel 0.21
+authors:
+  - laurentlb
+---
+
+We've just released [Bazel 0.21](https://github.com/bazelbuild/bazel/releases/tag/0.21.0)!
+
+We also revamped our Android with Bazel tutorial. Check it out [here](https://docs.bazel.build/versions/master/tutorial/android-app.html).
```
Comment (laszlocsomor) [inline _posts/2018-12-19-bazel-0.21.md]: What's the reason for having moved this out of the Android section?

File: _posts/2018-12-19-bazel-0.21.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+*   [`repository_ctx.download`](https://docs.bazel.build/versions/master/skylark/lib/repository_ctx.html#download) and [`repository_ctx.download_and_extract`](https://docs.bazel.build/versions/master/skylark/lib/repository_ctx.html#download_and_extract) now return a struct, instead of `None`.
+
+
+## Execution
+
+*   `--genrule_strategy` is deprecated in favor of [`--strategy=Genrule=<strategy>`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--strategy) (for genrules) or [`--spawn_strategy=<strategy>`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--spawn_strategy) (for all actions).
+*   Dynamic execution is now available with [`--experimental_spawn_scheduler`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--experimental_spawn_scheduler). Dynamic execution allows a build action to run locally and remotely simultaneously, and Bazel picks the fastest action. This provides the best of both worlds: faster clean builds than pure local builds, and faster incremental builds than pure remote builds.
+
+
+## Other changes
```
Comment (laszlocsomor) [inline _posts/2018-12-19-bazel-0.21.md]: Please also mention that the docs are now versioned, thanks to @jin 's much-appreciated efforts \o/

File: _posts/2018-12-19-bazel-0.21.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+are marked with GitHub label [migration-0.21](
+https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+label%3Amigration-0.21).
+
+Breaking changes in the next release (0.22) are marked with GitHub label
+[breaking-change-0.22](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+label%3Abreaking-change-0.22).
+
+
+## Android
+
+*   [`--experimental_check_desugar_deps`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--experimental_check_desugar_deps)
```
Comment (laszlocsomor) [inline _posts/2018-12-19-bazel-0.21.md]: All bazel.build links should point to the right version, not master.

File: _posts/2018-12-19-bazel-0.21.md [hunk 4]
```diff
@@ -104,7 +104,7 @@ Breaking changes in the next release (0.22) are marked with GitHub label
 
 ## Community
 
-*   Natan Silnitsky talked about Wix's [lessons learned for successful migration to Bazel and Google Cloud Build](https://www.youtube.com/watch?v=qAKSHpzpRvw) to build their Scala codebase.
+*   Natan Silnitsky talked about Wix's [lessons learned for successful migration to Bazel and Google Cloud Build](https://www.youtube.com/watch?v=nKx21CsT744) to build their Scala codebase.
 *   David Stanke also did a talk: [Build and test your code faster with Bazel, the cloud-accelerated build system](https://www.youtube.com/watch?v=NqRpf7rJvP4).
```
Comment (laszlocsomor) [inline _posts/2018-12-19-bazel-0.21.md]: This one is removed too :(

File: _posts/2018-12-19-bazel-0.21.md (hunk 1 above)
Comment (laurentlb) [reply _posts/2018-12-19-bazel-0.21.md]: I wanted to highlight it (also, it's not directly a change to Bazel).

Comment (laszlocsomor) [reply _posts/2018-12-19-bazel-0.21.md]: That implies to me that this announcement is more important than the others, and I don't see why.

File: _posts/2018-12-19-bazel-0.21.md (hunk 3 above)
Comment (laurentlb) [reply _posts/2018-12-19-bazel-0.21.md]: It's not available.
https://docs.bazel.build/versions/0.21.0/ doesn't exist yet.
We can improve our release workflow for 0.22

**Pull request merged**